src/store/store.tsx

```diff
@@ -4,19 +4,22 @@ import { PropsWithChildren, useMemo } from "react";
 import { Provider as ReduxProvider } from "react-redux";
 import { createLogger } from "redux-logger";
 
-import ApiSlice, { ApiDataStore, apiSlice, authListenerMiddleware } from "@/store/apiSlice";
+import ApiSlice, { ApiDataStore, apiSlice, authListenerMiddleware } from "./apiSlice";
+import JobsSlice, { JobsDataStore, jobsSlice } from "./jobsSlice";
 
 // Action used only in test suites to dump some specific state into the store.
 export const __TEST_HYDRATE__ = "__TEST_HYDRATE__";
 
 export type AppStore = {
   api: ApiDataStore;
+  jobs: JobsDataStore;
 };
 
 export const makeStore = (queryClient?: QueryClient) => {
   const store = configureStore({
     reducer: {
-      api: apiSlice.reducer
+      api: apiSlice.reducer,
+      jobs: jobsSlice.reducer
```

Comment:
As you can see here, it's really easy to create isolated "slices" of the redux store. Next time we have some globally accessible state we want to share between lots of places in the app stack / component tree, spinning up a new slice is greatly preferable to slotting them into an existing one. Nice separation of concerns this way.